data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+		system
+			distance 15 20
+		fleet "Large Northern Pirates"
+	on stopover
+		dialog `You have placed cubes on the last of the planets that Paris has asked you to. When you look into the bag you find that there are three left. Go to <destination> to give the remaining cubes to Paris.`
+	on complete
+		event "deep: warlord detected" 20 30
+		payment 500000
+		conversation
+			`You find Lieutenant Paris waiting for you in the spaceport. You hand him the bag of remaining cubes, and he hands you <payment> for helping.`
+			
+			branch interested
+				has "deep: interested in cubes"
+			
+			choice
+				`	"Is there anything else that you need me to do?"`
+					goto next
+				`	"What is it that those cubes are for?"`
+					goto cubes
+			
+			label interested
+			choice
+				`	"Is there anything else that you need me to do?"`
+					goto next
+				`	"Are you able to tell me what the cubes are for now?"`
+			
+			label cubes
+			apply
+				set "deep: interested in cubes"
+			`	Paris takes a moment to pause before responding. "They're sensor devices. They are used to track the movement of pirate fleets, but they are almost useless alone; dozens could be needed in a single system to track entire fleets. The ones that I gave you were fine-tuned though, designed to only detect the ship that we know that Beelzebub is using. That is all that I am allowed to tell you, at least for the time being."`
+			
+			choice
+				`	"Is there anything else that you need me to do?"`
+			
+			label next
+			`	"Not at the moment, Captain, but I'll contact you should anything else arise."`
+
+event "deep: warlord detected"
+
+
+
+mission "Deep: Mystery Cubes 1"
+	minor
+	landing
+	name `Travel to <planet>`
+	description `Travel to <destination> to learn more about the situation with the warlord Beelzebub.`
+	source
+		government "Republic" "Free Worlds" "Neutral" "Syndicate" "Independent"
+		near "Sol" 1 100
+	destination "Luna"
+	to offer
+		has "event: deep: warlord detected"
+	
+	on offer
+		dialog `You receive a message from Lieutenant Paris. "Greetings, Captain. The situation involving Beelzebub has gone slightly sour, and the Navy is involved now. I have been asked to gather the merchants who assisted the Deep previously in placing the cubes. If you are interested, please come to <destination>."`
+
+
+
+mission "Deep: Mystery Cubes 2"
+	landing
+	name `Blockading the Devil`
+	description `Travel to <destination> to set up a blockade in homes of capturing the warlord Beelzebub.`
+	destination "Maelstrom"
+	deadline
+	to offer
+		has "Deep: Mystery Cubes 1: done"
+	
+	on offer
+		conversation
+			`Upon landing, you are escorted by a pair of men in Navy uniforms past the tourist areas of the Moon facility and into the back to an elevator. One of your escorts uses a keycard to open the elevator, ushering you in. The elevator opens up many floors below the spaceport, to an area of the facility that looks far newer than that on the surface.`
```

Comment:
Recommend changing this line to:
```After you land, a pair of men in Navy uniforms escort you to an elevator past the tourist areas of the Moon facility. One of your escorts uses a keycard to open the elevator and ushers you in. The elevator opens up many floors below the spaceport, to an area of the facility that looks far newer than that on the surface.```

The converts the first sentence to active voice and makes it a bit shorter. It also changes a use of the present progressive ("ushering") to match the rest of the section.